kafka/adminapi_test.go

```diff
@@ -30,26 +30,30 @@ func TestAdminAPIWithDefaultValue(t *testing.T) {
 		t.Skipf("Missing testconf.json")
 	}
 
-	topic := "testWithDefaultValue"
-
 	conf := ConfigMap{"bootstrap.servers": testconf.Brokers}
 	if err := conf.updateFromTestconf(); err != nil {
 		t.Fatalf("Failed to update test configuration: %v\n", err)
 	}
 
-	expDuration, err := time.ParseDuration("30s")
+	adminClient, err := NewAdminClient(&conf)
 	if err != nil {
-		t.Fatalf("Failed to Parse Duration: %s", err)
+		t.Fatalf("Failed to create AdminClient %v", err)
 	}
+	testAdminAPIWithDefaultValue(t, adminClient, nil)
+	testAdminAPIWithDefaultValue(t, adminClient, getOperationNotAllowedErrorForClosedClient())
```

Comment:
Add comment here that `adminClient` is closed by the time we call the second one, because otherwise it requires one to read the whole other function.